README.md

```diff
@@ -1,122 +1,75 @@
-
 ## Completing Authorization Flow
-
 NHS login will return an authorization code, which you then use to obtain the ID Token. This is done by making a request to the NHS login token endpoint. The token endpoint requires authentication via private_key_jwt  [method](https://openid.net/specs/openid-connect-core-1_0.html#ClientAuthentication). The returned ID Token must be validated before it can be used within your service.
-
 # How can I test NHS login in my service?
-
 We have a stub and a sandpit that development teams can use to familiarise themselves with NHS login.
-
 ## NHS login stub
-
 We have developed a lightweight stub that is a representation of our OIDC endpoints. You can find out more about the stub and access it here: [https://stub.signin.nhs.uk/client](https://stub.signin.nhs.uk/client). The stub is entirely self-service and requires no registration.
-
 ## Sandpit
-
 Our sandpit is a deployment of much of the production code, with some elements of the registration process removed e.g. matching to NHS number and checking ID documents.
-
 # How do I integrate to the sandpit?
-
 We’ll need to setup your service on NHS login sandpit. Please provide the below information to [engage.nhslogin@nhs.net](mailto:engage.nhslogin@nhs.net?subject=Request%20Sandpit%20Access)
- 
 Required information:
-
 -   A friendly name of your service e.g. what the public know you as
-    
 -   Your redirect_uri that we’ll return traffic to
-    
 -   Your public key for validating signed JWTs - see below for guidance on generating this
-
-  
 We aim to process these requests within 24 hours.
-
-
 You will need to add an OIDC Relying Party library to your service. OpenID certified examples can be found here: [https://openid.net/developers/certified/](https://openid.net/developers/certified/)
-
 We have provided links to a couple of example repos at the bottom of this file. We aim to add more client examples over the coming months. If you have one to add, please raise an issue or PR.
-
 Depending on the library you pick you’ll either need:
-
 -   the configuration endpoint: [https://auth.sandpit.signin.nhs.uk/.well-known/openid-configuration](https://auth.sandpit.signin.nhs.uk/.well-known/openid-configuration)
-    
 -   or the specific endpoints, which can be found at the configuration endpoint
-
-## Testing the login flow in the Sandpit
+## Testing the login flow
 Before testing the registration flow you should ensure a login flow can be completed. This involves all of the interaction points, such as the token and userinfo endpoints.
-
 There are a number of users already configured that you can use to test the login flow.   We'll send you the test pack when we configure your client.
-
 ## Testing the registration process
 We currently have two methods to complete registration:
-
 - Existing GP Online details
 - Online ID check with video selfie
-
-We do not support testing of GP Online method in the sandpit, but you can test the online ID checking process.
```

Comment:
Removal of line space flattens this section, which doesn't read well.

Reply:
this has been updated now